src/components/Navigation/SharedStyles.js

```diff
@@ -143,12 +125,7 @@ export const NavItem = styled.div`
 `;
 
 export const StickyEl = styled.div`
-  width: 100%;
   height: 100vh;
-  /* --vh will be set via setViewportHeight() ApiReference.js */
-  height: calc(var(--vh, 1vh) * 100);
-  overflow-y: scroll;
-  position: sticky;
-  top: 0;
-  z-index: 3;
+  display: flex;
```

Comment:
I changed out how this element's layout is done from absolute to flexbox, think it's a little more stable now